services/moneypenny/values-idfdev.yaml

```diff
@@ -0,0 +1,41 @@
+moneypenny:
+  host: "data-dev.lsst.cloud"
+
+  ingress:
+    enabled: true
+    hosts:
+      - host: data-dev.lsst.cloud
+        paths: ["/moneypenny"]
+    annotations:
+      nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Uid, X-Auth-Request-Token
```

Comment:
I am very dubious that you're using this, given that this is synthesized data from your fake token and you're passing the UID you care about as the request body. I think you want to delete this.